.drone.yml

```diff
@@ -0,0 +1,140 @@
+---
+kind: pipeline
+name: linting
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: later
+  image: toolhippie/ansible-later:latest
+  pull: always
+  commands:
+    - ansible-later
+
+trigger:
+  ref:
+  - refs/heads/master
+  - refs/tags/**
+  - refs/pull/**
+
+---
+kind: pipeline
+name: deployment
+
+platform:
+  os: linux
+  arch: amd64
+
+concurrency:
+  limit: 1
+
+workspace:
```

Comment:
Why do we need a workspace here?

Reply:
Because I have not found a better way. Molecule does not find the role otherwise.